src/app/homeseeker/page.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button, Card } from "@tremor/react";
+import Link from "next/link";
+import { NextPage } from "next";
+import { getProperties } from "@/db/homeseeker/property";
+import { getCurrentUser } from "@/app/cookies";
+
+export async function filterPropertiesByUser() {
+	const properties = await getProperties();
+	const user = getCurrentUser();
+	// Not signed it so just display everything
+	if (!user) {
+		return properties;
+	}
+	const filteredProperties = properties.filter((property) => {
+		return property.broker_id !== user.id
+	})
+	return filteredProperties;
```

Comment:
Perhaps this should be an option? Or maybe you can show a little badge on listings that are posted by the current user? I feel like hiding *all other listings* when you're signed in is a bit weird.

Reply:
I was thinking if this is a large scale home seeker app, it doesn't make sense for user to view their own homes. They would want to see other property listed. For now they can still view their registered properties in their account page. But maybe you're right we can just add something to show that the property is owned by a certain user instead of filtering them.